### ISSUE TITLE
Inconsistent parameter naming Macro __DEFINE_HAS_LVAL_MMBR_FUNC_ 6

### ISSUE THREAD

**Issue by AnantaYudica** [AnantaYudica/basic#14]
```
#include "macro/HasLValueMemberFunction.h"

template<typename>
struct C
{
    void Foo1() {}
};

/**
 *  template<typename T1, typename Tr, typename... Targs,
 *      template<typename> class T = A>
 *  constexpr auto _HasLValMmbrFunc6(T<T1> t, Targs... args) ->
 *      decltype(Tr(t.Foo1(args...)), std::true_type());
 *  template<typename T, typename Tr, typename... Targs>
 *  constexpr std::false_type _HasLValMmbrFunc6(...);
 *  template<typename T, typename Tr, typename... Targs>
 *  struct HasLValMmbrFunc6
 *  {
 *      typedef bool value_type;
 *      typedef bool ValueType;
 *      static constexpr value_type value = decltype(_HasLValMmbrFunc6<
 *          Tr, Targs...>(
 *              std::declval<typename std::remove_cv<typename basic::type::
 *                  trait::rem::ptr::Recursive<T>::type>::type>(), 
 *              std::declval<typename std::remove_cv<Targs...>::type>()
 *          ))::value;
 *      static constexpr ValueType Value = decltype(_HasLValMmbrFunc6<
 *          Tr, Targs...>(
 *              std::declval<typename std::remove_cv<typename basic::type::
 *                  trait::rem::ptr::Recursive<T>::type>::type>(), 
 *              std::declval<typename std::remove_cv<Targs...>::type>()
 *          ))::value;
 *  };
 * */

#define __DEFINE_HAS_LVAL_MMBR_FUNC_PROTO_FUNC_TRUE_TMPL_PARAM_LIST_PFTTPL2(\
    __T_NAME__,\
    __T_RET_NAME__,\
    __T_ARGS_NAME__)\
typename __T_RET_NAME__, typename... __T_ARGS_NAME__, typename T1,\
    template<typename> class __T_NAME__ = C

#define __DEFINE_HAS_LVAL_MMBR_FUNC_PROTO_FUNC_FALSE_TMPL_PARAM_LIST_PFFTPL2(\
    __T_NAME__,\
    __T_RET_NAME__,\
    __T_ARGS_NAME__)\
typename __T_RET_NAME__, typename... __T_ARGS_NAME__

#define __DEFINE_HAS_MMBR_FUNC_PROTO_FUNC_T_NAME_TMPL_ARG_LIST_FTNTAL1()\
T1

#define __DEFINE_HAS_LVAL_MMBR_FUNC_PROTO_FUNC_TMPL_ARG_LIST_PFTAL3(\
    __T_NAME__, \
    __T_RET_NAME__,\
    __T_ARGS_NAME__,\
    __T_NAME_REM_RECV_PTR_TYPE__, ...)\
__T_RET_NAME__, __T_ARGS_NAME__...

__DEFINE_HAS_LVAL_MMBR_FUNC_(HasLValMmbrFunc6, _HasLValMmbrFunc6, T, Tr, 
    Targs,, T_NAME_,, PFTTPL2, PFFTPL2, PFTAL3, PROTO_FUNC_T_NAME_TMPL_, FTNTAL1,\
    PROTO_FUNC_T_RET_NAME_,, Foo1);
```
parameter macro using name `__PROTO_FUNC_T_NAME_TMPL_ARG_LIST_POSTFIX__` in `__DEFINE_HAS_LVAL_MMBR_FUNC_`
but call back implementation using name `__DEFINE_HAS_MMBR_FUNC_PROTO_FUNC_T_NAME_TMPL_ARG_LIST_`

Similar with #4, #9, #10, #11, #12 and #13

**Comment by AnantaYudica** [comment]
Fixed in v0.1-beta-2

**Comment by AnantaYudica** [comment]
Not a bug, Macro `__DEFINE_HAS_LVAL_MMBR_FUNC_` will be deleted in v0.1-beta-2.
